manim/mobject/value_tracker.py

```diff
@@ -37,8 +37,14 @@ def construct(self):
                 )
                 self.add(number_line, pointer,label)
                 self.play(pointer_value.animate.set_value(5)),
-                self.wait()
+                self.wait(0.5)
                 self.play(pointer_value.animate.set_value(3))
+                self.play(pointer_value.animate.increment_value(-2))
+                self.wait(0.5)
+                pointer_value += 1.5
+                self.wait(1)
+                pointer_value -= 4
+                self.wait(0.5)
```

Comment:
I think increasing the wait times would make the animation loop more smoothly.
```py
self.play(pointer_value.animate.increment_value(-2))
self.wait(1)
pointer_value += 1.5
self.wait(1)
pointer_value -= 4
self.wait(2)
```

Reply:
As pointer_value += 1.5 is a sudden jump, I think waiting  `self.wait(0.5)`is enough.
We also should keep in mind when making the docs example, that each additional frame will increase the overall building time of the documentation, as every time we make a commit, the whole docs (including all examples) are rendered from scratch.

Reply:
I don't think adding longer `wait`s increases the doc build time substantially (if it increases them at all).

Reply:
Okay.
I actually prefer the shorter time: https://manimce--1246.org.readthedocs.build/en/1246/reference/manim.mobject.value_tracker.ValueTracker.html?highlight=valuetracker#valuetrackerexample
But that is personal taste.
@RickyC0626 : If you think it is a lot better with longer time, feel free to push a commit, but I prefer the current solution.